content/en/docs/ibm/pipelines.md

```diff
@@ -34,3 +79,75 @@ experiments = client.list_experiments(namespace=KUBEFLOW_PROFILE_NAME)
 ```
 
 Pipelines components like experiments and runs are isolated by Kubeflow profiles. A Kubeflow user can only see Pipelines experiments and runs belonging to this user's Kubeflow profile.
+
+# Upload pipelines
+
+Once you have declared the Python client, your Kubeflow pipelines can be uploaded using Python. 
+
+Run the following code inside a Python session to upload the pipelines. This example shows different versions of the pipeline using the Python client.
+
+```python
+import os
+
+# Initial version of the compiled pipeline
+pipeline_file_path = 'echo_pipeline.yaml'
+pipeline_name = 'echo_pipeline'
+
+# For the purpose of this tutorial, we will be using the same pipeline for both version.
+pipeline_version_file_path = 'echo_pipeline.yaml'
+pipeline_version_name = 'new_echo_pipeline'
+
+# Upload initial version of the pipeline
+pipeline_file = os.path.join(pipeline_file_path)
+pipeline = client.pipeline_uploads.upload_pipeline(pipeline_file, name=pipeline_name)
+
+# Upload new version of the pipeline
+pipeline_version_file = os.path.join(pipeline_version_file_path)
+pipeline_version = client.pipeline_uploads.upload_pipeline_version(pipeline_version_file,
+                                                                   name=pipeline_version_name,
+                                                                   pipelineid=pipeline.id)
+```
+
+# Run pipelines from the SDK
+The `TektonClient` can run pipelines using one of the below sources:
+
+1. [Python DSL source code](#run-pipelines-from-the-python-dsl-source-code)
+2. [Compiled pipeline file](#run-pipelines-from-the-compiled-pipeline-file)
+3. [List of uploaded pipelines](#run-pipelines-from-the-list-of-uploaded-pipelines)
```

Comment:
```suggestion
3. [Uploaded pipelines on KFP engine](#run-pipelines-from-the-list-of-uploaded-pipelines)
```

Reply:
thanks @animeshsingh for the great suggestions, learning a lot here